webapp/pom.xml

```diff
@@ -84,17 +91,18 @@
       <artifactId>micrometer-registry-prometheus</artifactId>
     </dependency>
 
-    <!-- observability with Otel -->
+    <!-- Core OpenTelemetry dependencies -->
     <dependency>
-      <groupId>io.opentelemetry.javaagent</groupId>
-      <artifactId>opentelemetry-javaagent</artifactId>
-      <version>${opentelemetry.version}</version>
-      <scope>provided</scope>
+      <groupId>io.opentelemetry.instrumentation</groupId>
+      <artifactId>opentelemetry-spring-boot-starter</artifactId>
     </dependency>
     <dependency>
-      <groupId>io.opentelemetry.instrumentation</groupId>
-      <artifactId>opentelemetry-instrumentation-annotations</artifactId>
-      <version>${opentelemetry.version}</version>
+      <groupId>io.opentelemetry</groupId>
+      <artifactId>opentelemetry-exporter-otlp</artifactId>
```

Comment:
Is the `opentelemetry-javaagent` pulled when including this one? I thought your proposition was to keep the Java agent and have it still enabled for the generic use case?